kubetest/objects/persistentvolume.py

```diff
@@ -0,0 +1,100 @@
+"""Kubetest wrapper for the Kubernetes ``PersistentVolume`` API Object."""
+
+import logging
+
+from kubernetes import client
+
+from .api_object import ApiObject
+
+log = logging.getLogger('kubetest')
+
+
+class PersistentVolume(ApiObject):
+    """Kubetest wrapper around a Kubernetes `PersistentVolume`_ API Object.
+
+    The actual ``kubernetes.client.V1PersistentVolume`` instance that this
+    wraps can be accessed via the ``obj`` instance member.
+
+    This wrapper provides some convenient functionality around the
+    API Object and provides some state management for the `PersistentVolume`_.
+
+    .. _PersistentVolume:
+        https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#persistentvolume-v1-core
+    """
+
+    obj_type = client.V1PersistentVolume
+
+    api_clients = {
+        'preferred': client.CoreV1Api,
+        'v1': client.CoreV1Api,
+    }
+
+    def __str__(self):
+        return str(self.obj)
+
+    def __repr__(self):
+        return self.__str__()
+
+    def create(self):
+        """Create the PersistentVolume under the given namespace.
+
+        Args:
+            namespace (str): The namespace to create the PersistentVolume under.
+                If the PersistentVolume was loaded via the kubetest client, the
+                namespace will already be set, so it is not needed here.
+                Otherwise, the namespace will need to be provided.
```

Comment:
Need to update the docstring:

- Remove `Args` block, as there is no namespace arg here
- Update summary line, removing `under the given namespace`, since no namespace is given.